chunk 1: Tweak page headers (local change).
chunk 2: Default value for the .Os macro (local change).
chunk 3: Do not use an ugly double dash in the NAME line (local change).
chunk 4: Set the base paragraph indentation to 5n (upstream commit 5d2e49f8).

Index: tmac/mdoc/doc-common
--- tmac/mdoc/doc-common.orig
+++ tmac/mdoc/doc-common
@@ -335,7 +335,7 @@
 .ds doc-volume-ds-1 General Commands Manual
 .ds doc-volume-ds-2 System Calls Manual
 .ds doc-volume-ds-3 Library Functions Manual
-.ds doc-volume-ds-4 Kernel Interfaces Manual
+.ds doc-volume-ds-4 Device Drivers Manual
 .ds doc-volume-ds-5 File Formats Manual
 .ds doc-volume-ds-6 Games Manual
 .ds doc-volume-ds-7 Miscellaneous Information Manual
@@ -490,7 +490,7 @@
 .\" NS
 .\" NS override this in 'mdoc.local', if necessary
 .
-.ds doc-default-operating-system GNU\"
+.ds doc-default-operating-system OpenBSD\"
 .
 .
 .\" NS doc-operating-system global string
@@ -1244,7 +1244,7 @@
 .
 .eo
 .de Nd
-.  nop \[em] \$*
+.  nop \[en] \$*
 ..
 .ec
 .
@@ -1482,7 +1482,7 @@
 .    if t \
 .      ss \n[doc-reg-Sh] \n[doc-reg-Sh1]
 .
-.    in \n[IN]u
+.    in \n[BP]u
 .    ns
 .
 .    doc-check-depth
